dev/com.ibm.ws.microprofile.metrics.monitor_fat/fat/src/com/ibm/ws/microprofile/metrics/monitor_fat/TestEnableDisableFeaturesTest.java

```diff
@@ -0,0 +1,680 @@
+        if (server != null && server.isStarted()) {
+            server.stopServer("CWWKS4000E", "CWWKZ0014W");
+            server.removeAllInstalledAppsForValidation();
+        }
+    }
+    
+    @BeforeClass
+    public static void setUp3Apps() throws Exception {
+    	add3Apps(serverEDF6);
+    	add3Apps(serverEDF8);
+    	add3Apps(serverEDF9);
+    	add3Apps(serverEDF10);
+    }
+    
+    public static void add3Apps(LibertyServer server) throws Exception {
+    	String testName = "add3Apps to " + server.toString();
+    	Log.info(c, testName, "------- Add session application ------");
+       	ShrinkHelper.defaultDropinApp(server, "testSessionApp", "com.ibm.ws.microprofile.metrics.monitor_fat.session.servlet");
+       	Log.info(c, testName, "------- added testSessionApp to dropins -----");
+       	Log.info(c, testName, "------- Add JDBC application ------");
+       	ShrinkHelper.defaultDropinApp(server, "testJDBCApp", "com.ibm.ws.microprofile.metrics.monitor_fat.jdbc.servlet");
+       	Log.info(c, testName, "------- added testJDBCApp to dropins -----");
+    	Log.info(c, testName, "------- Add jax-ws endpoint application ------");
+       	ShrinkHelper.defaultDropinApp(server, "testJaxWsApp", "com.ibm.ws.microprofile.metrics.monitor_fat.jaxws","com.ibm.ws.microprofile.metrics.monitor_fat.jaxws.client");
+       	Log.info(c, testName, "------- added testJaxWsApp to dropins -----");
+    }
+    
+    @BeforeClass
+    public static void setUp2Apps() throws Exception {
+    	add2Apps(serverEDF11);
+    	add2Apps(serverEDF5);
+    }
+     
+    public static void add2Apps(LibertyServer server) throws Exception {
+    	String testName = "add2Apps to " + server.toString();
+    	Log.info(c, testName, "------- Add session application ------");
+       	ShrinkHelper.defaultDropinApp(server, "testSessionApp", "com.ibm.ws.microprofile.metrics.monitor_fat.session.servlet");
+       	Log.info(c, testName, "------- added testSessionApp to dropins -----");
+       	Log.info(c, testName, "------- Add JDBC application ------");
+       	ShrinkHelper.defaultDropinApp(server, "testJDBCApp", "com.ibm.ws.microprofile.metrics.monitor_fat.jdbc.servlet");
+       	Log.info(c, testName, "------- added testJDBCApp to dropins -----");
+    }
+    
+    @BeforeClass
+    public static void setUpEDF12() throws Exception {
+    	String testName = "setUpEDF12";
+    	Log.info(c, testName, "------- Add session application ------");
+       	ShrinkHelper.defaultDropinApp(serverEDF12, "testSessionApp", "com.ibm.ws.microprofile.metrics.monitor_fat.session.servlet");
+       	Log.info(c, testName, "------- added testSessionApp to dropins -----");
+    }
+    
+    @Test
+    public void testEDF1() throws Exception {
+
+    	String testName = "testEDF1";
+        Log.info(c, testName, "------- No monitor-1.0: no vendor metrics should be available ------");
+        serverEDF1.setServerConfigurationFile("server_mpMetric11.xml");
```

Comment:
A lot of test cases set the server.xml right at the beginning.  Can we simply start the server with this server.xml rather than setting it in the testcase?